pandas/_libs/tslib.pyx

```diff
@@ -101,7 +105,8 @@ def format_array_from_datetime(
     ndarray[int64_t] values,
```

Comment:
if a tslibs.strftime does get implemented, that might be a good home for format_array_from_datetime